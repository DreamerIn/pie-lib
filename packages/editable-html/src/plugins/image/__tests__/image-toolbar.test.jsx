import React from 'react';
import Toolbar from '../image-toolbar';
import renderer from 'react-test-renderer';
import { Data, Block, Value } from 'slate';

it('renders correctly', () => {
  const node = Block.fromJSON({
    type: 'image',
    data: Data.create({
      resizePercent: 100
    })
  });

  const classes = { holder: 'holder' };

  const tree = renderer
    .create(
      <Toolbar
        node={node}
        classes={classes}
        value={Value.fromJSON({})}
        onChange={jest.fn()}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
