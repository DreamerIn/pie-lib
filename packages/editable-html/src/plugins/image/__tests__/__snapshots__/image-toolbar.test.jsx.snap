// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="holder"
>
  <span
    className="RawMarkButton-button-0-1-2"
    onMouseDown={[Function]}
  >
    25%
  </span>
  <span
    className="RawMarkButton-button-0-1-2"
    onMouseDown={[Function]}
  >
    50%
  </span>
  <span
    className="RawMarkButton-button-0-1-2"
    onMouseDown={[Function]}
  >
    75%
  </span>
  <span
    className="RawMarkButton-button-0-1-2 RawMarkButton-active-0-1-3"
    onMouseDown={[Function]}
  >
    100%
  </span>
</div>
`;
